src/FacultyDirectory/Controllers/UsersController.cs

```diff
@@ -45,50 +31,39 @@ public ActionResult Get()
             return Json(userData);
         }
 
-        // should be POST
-        [HttpGet("validate")]
-        public ActionResult Validate() {
-            // TODO: get auth header
-            var jwt = "jwt";
-
-            var key = Encoding.ASCII.GetBytes("secretstringhere-verycoolsecret-thebest");
-            var sharedKey = new SymmetricSecurityKey(key);
+        [HttpGet]
+        public async Task<ActionResult> GetUsers() {
+            return Ok(await dbContext.Users.ToArrayAsync());
+        }
 
-            var validationParameters = new TokenValidationParameters
-            {
-                // Clock skew compensates for server time drift.
-                // We recommend 5 minutes or less:
-                ClockSkew = TimeSpan.FromMinutes(5),
-                // Specify the key used to sign the token:
-                IssuerSigningKeys = new [] { sharedKey },
-                RequireSignedTokens = true,
-                // Ensure the token hasn't expired:
-                RequireExpirationTime = true,
-                ValidateLifetime = true,
-                ValidateAudience = false,
-                ValidateIssuer = false
+        [HttpPost]
+        public async Task<ActionResult> AddUser(User userData) {
+            // Creates a user
+            var user = new User 
```

Comment:
minor, and optional, but it might be worth checking to see if that username is already in the db before adding it again.